integration/js/pages/AppPage.js

```diff
@@ -31,7 +31,7 @@ function findAllElements() {
     dataMessageSendInput: By.id('send-message'),
     sipAuthenticateButton: By.id('button-sip-authenticate'),
     roster: By.id('roster'),
-    participants: By.css('li'),
+    participants: By.css('#roster>li'),
```

Comment:
Hopefully this will not break prod canary (which I updated recently to point to the last release branch). Previously it will break as the prod stage uses the demo deployed when NPM published.